[2024-10-29T00:13:58.581+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-29T00:13:58.629+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: indices_provincias_v01.inflacion_santafe scheduled__2024-10-28T00:00:00+00:00 [queued]>
[2024-10-29T00:13:58.649+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: indices_provincias_v01.inflacion_santafe scheduled__2024-10-28T00:00:00+00:00 [queued]>
[2024-10-29T00:13:58.650+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-10-29T00:13:58.681+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): inflacion_santafe> on 2024-10-28 00:00:00+00:00
[2024-10-29T00:13:58.708+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'indices_provincias_v01', 'inflacion_santafe', 'scheduled__2024-10-28T00:00:00+00:00', '--job-id', '252', '--raw', '--subdir', 'DAGS_FOLDER/second_dag.py', '--cfg-path', '/tmp/tmpr92zbibn']
[2024-10-29T00:13:58.723+0000] {standard_task_runner.py:105} INFO - Job 252: Subtask inflacion_santafe
[2024-10-29T00:13:58.730+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=192) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-29T00:13:58.731+0000] {standard_task_runner.py:72} INFO - Started process 205 to run task
[2024-10-29T00:13:58.836+0000] {task_command.py:467} INFO - Running <TaskInstance: indices_provincias_v01.inflacion_santafe scheduled__2024-10-28T00:00:00+00:00 [running]> on host 276004bbea2b
[2024-10-29T00:13:59.301+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='rolo' AIRFLOW_CTX_DAG_ID='indices_provincias_v01' AIRFLOW_CTX_TASK_ID='inflacion_santafe' AIRFLOW_CTX_EXECUTION_DATE='2024-10-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-28T00:00:00+00:00'
[2024-10-29T00:13:59.302+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-29T00:14:35.925+0000] {logging_mixin.py:190} INFO - Error: [CANNOT_INFER_TYPE_FOR_FIELD] Unable to infer the type of the field `fecha_inflacion`.
[2024-10-29T00:14:36.200+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-29T00:14:36.214+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-29T00:14:36.215+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=indices_provincias_v01, task_id=inflacion_santafe, run_id=scheduled__2024-10-28T00:00:00+00:00, execution_date=20241028T000000, start_date=20241029T001358, end_date=20241029T001436
[2024-10-29T00:14:36.284+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-29T00:14:36.316+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-29T00:14:36.321+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
